cmd/derper/derper.go

```diff
@@ -468,3 +458,51 @@ func init() {
 		return 0
 	}))
 }
+
+type templateData struct {
+	ShowAbuseInfo bool
+	Disabled      bool
+	AllowDebug    bool
+}
+
+var homePageTemplate = template.Must(template.New("home").Parse(`<html><body>
```

Comment:
// homePageTemplate renders the home page using [templateData].

?

(I've been trying to document the data type that templates take lately.)